platform/ui/src/components/DateRange/DateRange.jsx

```diff
@@ -0,0 +1,159 @@
+/** REACT DATES */
+import 'react-dates/initialize';
+import 'react-dates/lib/css/_datepicker.css';
+import { DateRangePicker, isInclusivelyBeforeDay } from 'react-dates';
+import './DateRange.css';
+
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+
+import moment from 'moment';
```

Comment:
Is this a dependency of the 3rd party library, or our own? Trying to think if there's a good way to set this as a peer dependency or make it possible for the consumer to make this decision

Reply:
As react-dates uses `moment` objects for date values, I'm using `moment` as our dependency to create the custom Filter Presets (Today, Last 7 Days, Last 30 Days) and to render the customized month/year select (as described below).